### PR TITLE
Enable skipped test again for pilot suite on our istio fork

### PR DESCRIPTION
**Pull request by fjglira**
More information here: https://issues.redhat.com/browse/OSSM-6316

### CODE REVIEW

**Comment by fjglira** [comment]
@sridhargaddam FYI

**Comment by fjglira** [comment]
Hey @sridhargaddam we got some failures in the test run:
* TestTunnelingOutboundTraffic
* TestCustomGateway/helm
* TestCustomGateway/helm-simple

**Comment by fjglira** [comment]
/pj-rehearse pull-ci-openshift-service-mesh-istio-master-istio-integration-pilot

**Comment by fjglira** [comment]
/pj-rehearse pull-ci-openshift-service-mesh-istio-master-istio-integration-pilot

File: ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-master.yaml [hunk 1]
```diff
@@ -95,7 +95,7 @@ tests:
           cd /work && \
           entrypoint \
           prow/integ-suite-ocp.sh \
-          pilot 'TestGatewayConformance|TestCustomGateway|TestCNIRaceRepair|TestGateway/managed-owner|TestProxyHeaders|TestCustomGateway/helm|TestCustomGateway/helm-simple|TestTcpProbe/rewrite-success|TestGRPCProbe|TestTunnelingOutboundTraffic|TestCNIVersionSkew|TestGateway|TestIngress|TestLabelChanges|TestLocality|TestMirroring/mirror-percent-absent|TestMirroring/mirror-50|TestMirroring/mirror-10|TestMirroringExternalService'"
+          pilot 'TestGatewayConformance|TestCNIRaceRepair|TestCNIVersionSkew'"
```
Comment (sridhargaddam) [inline ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-master.yaml:98]: The [PR which addresses](https://github.com/istio/istio/pull/52974) `TestCustomGateway` is under review on `master` branch. So, lets skip them until its merged.

i.e., 
```
TestCustomGateway
TestCustomGateway/helm
TestCustomGateway/helm-simple
```

In my testing, I found the following tests as flaky. Would be better to skip them for now.

```
TestMirroring/mirror-50
TestMirroring/mirror-10
```

`TestMirroring/mirror-percent-absent` and `TestMirroringExternalService` was passing in my tests.

File: ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-release-1.23.yaml [hunk 2]
```diff
@@ -90,7 +90,7 @@ tests:
           cd /work && \
           entrypoint \
           prow/integ-suite-ocp.sh \
-          pilot 'TestGatewayConformance|TestCustomGateway|TestCNIRaceRepair|TestGateway/managed-owner|TestProxyHeaders|TestCustomGateway/helm|TestCustomGateway/helm-simple|TestTcpProbe/rewrite-success|TestGRPCProbe|TestTunnelingOutboundTraffic|TestCNIVersionSkew|TestGateway|TestIngress|TestLabelChanges|TestLocality|TestMirroring/mirror-percent-absent|TestMirroring/mirror-50|TestMirroring/mirror-10|TestMirroringExternalService'"
+          pilot 'TestGatewayConformance|TestCNIRaceRepair|TestCNIVersionSkew'"
```
Comment (sridhargaddam) [inline ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-release-1.23.yaml:93]: The [PR which addresses](https://github.com/istio/istio/pull/52974) `TestCustomGateway` is under review on `master` branch. So, lets skip them until its merged.

i.e., 
```
TestCustomGateway
TestCustomGateway/helm
TestCustomGateway/helm-simple
```

The PR for `TestTunnelingOutboundTraffic` test is merged on master, but its under review on the [1.23 branch](https://github.com/istio/istio/pull/53049), so lets skip it for now on this branch.

In my testing, I found the following tests as flaky. Would be better to skip them for now.

```
TestMirroring/mirror-50
TestMirroring/mirror-10
```

`TestMirroring/mirror-percent-absent` and `TestMirroringExternalService` was passing in my tests.

**Review by sridhargaddam — COMMENTED** [review]
Thanks for the PR @fjglira

**Pull request closed without merging**